src/config/sections.config.js

```diff
@@ -4,18 +4,17 @@ export const STANDARD_REPORT_SECTION_KEY = 'standard-report'
 export const DATA_SET_REPORT_SECTION_KEY = 'data-set-report'
 export const REPORTING_RATE_SUMMARY_SECTION_KEY = 'reporting-rate-summary'
 export const RESOURCE_SECTION_KEY = 'resource'
-export const ORGANISATION_UNIT_DISTRIBUTION_REPORT_SECTION_KEY =
+export const ORG_UNIT_DIST_REPORT_SECTION_KEY =
     'organisation-unit-distribution-report'
-export const ORGANISATION_UNIT_DISTRIBUTION_REPORT_SECTION_PATH =
-    '/organisation-unit-distribution-report'
+export const ORG_UNIT_DIST_REPORT_SECTION_PATH = `/${ORG_UNIT_DIST_REPORT_SECTION_KEY}`
```

Comment:
is there any reason to still keep this one? We can just use `sections[ORG_UNIT_DIST_REPORT_SECTION_KEY].path` where we need it, otherwise it's duplicate code.
If you want to keep that const, I recommend to do this:
```js
const { path: ORG_UNIT_DIST_REPORT_SECTION_PATH } = sections[ORG_UNIT_DIST_REPORT_SECTION_KEY]
```